contrib/generate_precompile.jl

```diff
@@ -99,10 +99,12 @@ function generate_precompile_statements()
                         pty_slave, pty_slave, pty_slave; wait=false)
                 readuntil(pty_master, "julia>", keep=true)
                 t = @async begin
+                    s = ""
```

Comment:
What's the point of this? Isn't this just renaming `s` -> `news` and adding a new unused `s` accumulating the string?

Reply:
I think it fixes the check `if occursin("__PRECOMPILE_END__", s)` below in case that string comes in multiple chunks.

Reply:
Ah, of course! Thanks!